modules/social_features/social_core/social_core.install

```diff
@@ -1693,7 +1693,7 @@ function social_core_update_11900() : string {
  * Permissions to check:
  * - "access contextual links".
  */
-function social_core_update_111101(): void {
```

Comment:
This change should land on 11.x as it it BC. Hook update number should not be changed.